src/components/Picklist-item.vue

```diff
@@ -1,32 +1,46 @@
 <template>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
-  </ion-item>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
-  </ion-item>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
-  </ion-item>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
+  <ion-item v-for="picklist in pickingList" :key="picklist.picklistId" @click="viewPicklist(picklist)" detail>
+    <ion-label>{{ user.partyName }}</ion-label>
```

Comment:
Is this information not available in picklist?

Reply:
No Sir, Picklist does not contain the pickers information.